### PR TITLE
Convert Team Settings layout to a NextJS layout

### PR DESCRIPTION
**Pull request by Oksamies**

### CODE REVIEW

**Comment by Oksamies** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/1032?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `master`
  * **PR #1023** <a href="https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/1023?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #1024** <a href="https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/1024?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #1029** <a href="https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/1029?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #1030** <a href="https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/1030?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #1031** <a href="https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/1031?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #1032** <a href="https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/1032?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/1032?utm_source=stack-comment).

File: apps/cyberstorm-nextjs/app/communities/layout.tsx [hunk 1]
```diff
@@ -1,19 +1,11 @@
 import "@thunderstore/cyberstorm-styles";
 import rootStyles from "../RootLayout.module.css";
-import { BreadCrumbs } from "@thunderstore/cyberstorm";
 import React from "react";
-import { PageHeader } from "@thunderstore/cyberstorm/src/components/Layout/BaseLayout/PageHeader/PageHeader";
 
 export default function CommunitiesLayout(props: React.PropsWithChildren) {
```
Comment (anttimaki) [inline apps/cyberstorm-nextjs/app/communities/layout.tsx:5]: This now looks like a generic layout that can be shared by all/most of the pages?

**Review by anttimaki — COMMENTED** [review]
Does this contain all the intended files? It's doesn't seem so much a conversion, but adding of new pages and leaving the old Layout-component to rot in @thunderstore/cyberstorm?

Other than that, comments given on https://github.com/thunderstore-io/thunderstore-ui/pull/1024 apply here too.

File: apps/cyberstorm-nextjs/app/communities/layout.tsx (hunk 1 above)
Comment (Oksamies) [reply apps/cyberstorm-nextjs/app/communities/layout.tsx:5]: Yeah, it kinda is. I'm still figuring out the NextJS layout structure. I'll refine the structure and "deduplicate" layouts as much as possible, after I've converted all the layouts to be under NextJS.

**Pull request merged**